spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/db2/V2_Add_Descriptions_And_OriginalDefinition.java

```diff
@@ -25,23 +25,26 @@
 
 /**
  * This migration class adds description column to stream_definitions and task_definitions
- * tables.
+ * tables and original_definition column to stream_definitions.
  *
  * @author Daniel Serleg
+ * @author Ilayaperumal Gopinathan
  */
-public class V2__Add_Descriptions extends BaseJavaMigration {
+public class V2_Add_Descriptions_And_OriginalDefinition extends BaseJavaMigration {
 
-	public final static String ALTER_STREAM_DEFINITION_TABLE = "alter table stream_definitions add description varchar(255)";
+	public final static String ALTER_STREAM_DEFINITION_TABLE_DESC = "alter table stream_definitions add description varchar(255)";
+	public final static String ALTER_STREAM_DEFINITION_TABLE_ORIG_DEF = "alter table stream_definitions add original_definition clob(255)";
 
-	public final static String ALTER_TASK_DEFINITION_TABLE = "" +
+	public final static String ALTER_TASK_DEFINITION_TABLE =
 			"alter table task_definitions add description varchar(255)";
 
 	private final SqlCommandsRunner runner = new SqlCommandsRunner();
 
 	@Override
 	public void migrate(Context context) throws Exception {
 		runner.execute(context.getConnection(), Arrays.asList(
-				SqlCommand.from(ALTER_STREAM_DEFINITION_TABLE),
+				SqlCommand.from(ALTER_STREAM_DEFINITION_TABLE_DESC),
+				SqlCommand.from(ALTER_STREAM_DEFINITION_TABLE_ORIG_DEF),
```

Comment:
When we're in a migration for an existing db, should we copy current definition to original? I know it's not original anymore but that's what we have. Otherwise that field is empty.

Reply:
Yeah, good point. I think we should.